### PR TITLE
Add kent's beginner course on react

### PR DESCRIPTION
**Pull request by elevenpassin**

### CODE REVIEW

**Comment by duttakapil** [comment]
I'll let @M-ZubairAhmed respond to this

File: web-dev/Frontend/Libraries & Frameworks/Learn-React.md [hunk 1]
```diff
@@ -371,3 +371,7 @@ React is an open source javaScript library for building user interfaces by Faceb
 | Concept       |                                                 Best Video Resource                                                 |                                         Best Text Resource                                         |                             Documentation                             | Duration |
 | :------------ | :-----------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------: | :------: |
 | Reconcilation | [Chariot Solutions](https://chariotsolutions.com/screencast/philly-ete-2016-30-jim-sproch-react-js-reconciliation/) | [Medium](https://medium.com/@gethylgeorge/how-virtual-dom-and-diffing-works-in-react-6fc805f9f84e) | [ReactJS/Reconcilation](https://reactjs.org/docs/reconciliation.html) | 10 hours |
+
+# Courses
+
+- (The Beginner's Guide To React)[https://egghead.io/courses/the-beginner-s-guide-to-react]
```
Comment (M-ZubairAhmed) [inline web-dev/Frontend/Libraries & Frameworks/Learn-React.md]: can you please check the typo here in rendering
image

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by M-ZubairAhmed** [comment]
@buoyantair i have another idea, why not have course too in a table,
with course name, provider, course duration. open to ideas here @duttakapil

File: web-dev/Frontend/Libraries & Frameworks/Learn-React.md (hunk 1 above)
Comment (elevenpassin) [reply web-dev/Frontend/Libraries & Frameworks/Learn-React.md]: @M-ZubairAhmed Good catch! I'll fix it!

**Comment by elevenpassin** [comment]
@M-ZubairAhmed a course? I wanted to include kent's work because its pretty neat and educational :D I did not find a good place to fit this in and I did not want to list each and every lesson from the course, so I put a courses section :3

**Comment by M-ZubairAhmed** [comment]
@buoyantair hmm ok makes sense

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**